src/get-own-property-symbols.js

```diff
@@ -93,7 +93,7 @@
       ));
     },
     Symbol = function Symbol(description) {
-      if (this && this !== G) {
+      if (this instanceof Symbol) {
```

Comment:
i assume this can't work cross-realm, but that's ok because it's a sham.

However, `Symbol.call(Symbol())` does not throw, so wouldn't this check make that fail?

Reply:
I added a testcase for this, it looks like it does not throw.

Reply:
I'm a bit confused; `function S() { if (this instanceof S) throw 42; return { __proto__: S.prototype }; } S.call(S())` throws. How is it possible that this doesn't?